README.md

```diff
@@ -177,19 +185,15 @@ and our resque log shows:
 *** SAMPLE_APP-RESQUE-00919a012476121cf89c done: (Job{default_queue} | EchoJob | ["hi from Rails console"] | {"log_weasel_id"=>"SAMPLE_APP-RESQUE-00919a012476121cf89c"})
 ```
 
-Units of work initiated from Resque, for example if using a scheduler like
-<a href="https://github.com/bvandenbos/resque-scheduler">resque-scheduler</a>,
-will include 'RESQUE' in the transaction ID to indicate that the work started in Resque.
+Units of work initiated from Resque, for example if using a scheduler like <a href="https://github.com/bvandenbos/resque-scheduler">resque-scheduler</a>, will include 'RESQUE' in the transaction ID to indicate that the work started in Resque.
 
 ## Contributing
 
-If you would like to contribute a fix or integrate Log Weasel transaction tracking into another frameworks
-please fork the code, add the fix or feature in your local project and then send a pull request on github.
-Please ensure that you include a test which verifies your changes.
+If you would like to contribute a fix or integrate Log Weasel transaction tracking into another frameworks please fork the code, add the fix or feature in your local project and then send a pull request on github. Please ensure that you include a test which verifies your changes.
 
 ## Authors
 
-<a href="http://github.com/asalant">Alon Salant</a> and <a href="http://github.com/brettfishman">Brett Fishman</a>.
+<a href="http://github.com/asalant">Alon Salant</a>, <a href="http://github.com/brettfishman">Brett Fishman</a>, and Rob Wierzbowski.
```

Comment:
I think I'd just assume get rid of the `Authors` section entirely. Alon and I created this library originally, but since, there have been many contributions from folks. Rather than including everyone here, how about we kill the section and let folks look at [contributors](https://github.com/stitchfix/log_weasel/graphs/contributors) if curious who the authors were.

Reply:
Yah, I did think it a little odd. Removing!